File: back-end/archiveq-parrent/document/src/main/java/com/pl/vkorol/document/repository/DocumentRepository.java
package com.pl.vkorol.document.repository;

import com.pl.vkorol.document.model.entity.DocumentInstance;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface DocumentRepository extends JpaRepository<DocumentInstance, String> {
}

File: back-end/search/src/main/kotlin/com/pl/vkorol/search/repository/DocumentRepository.kt
package com.pl.vkorol.search.repository;
import com.pl.vkorol.search.model.index.DocumentIndex
import org.springframework.data.elasticsearch.repository.ReactiveElasticsearchRepository;
import reactor.core.publisher.Flux

interface DocumentRepository : ReactiveElasticsearchRepository<DocumentIndex, String> {

    fun findByName(name: String): Flux<DocumentIndex>;
}
